doc/en/multitest.rst

```diff
@@ -1281,10 +1281,12 @@ The ``xfail`` decorator mandates a reason that explains why the test is marked a
     def fail_testcase(self, env, result):
         ...
 
-`Xpass` fails the testsuite/testcase, unless the ``strict`` parameter is passed as ``False``. A common example is a test is unstable:
+If a testcase/testsuite is unstable (passing sometimes, failling other times) then `strict=False` can be used. This means if the testcase/testsuite fails it woll be marked "expected to fail" (`xfail`), if it passes it will be marked as "unexpectedly passing" (`xpass`).
```

Comment:
I thought `strict=False` was the default option like it is in pytest? I don't think a user should need  to specify that explicitly.